### DISCUSSION TITLE
How to delete devices

### DISCUSSION THREAD

**Discussion by usama8800** [responsively-org/responsively-app#713] [Q&A] [answered]
How do I delete a custom device? Or even a default one

**Comment by manojVivek** [comment]
Currently, there is no option to do that. Can you please create an issue for this?

**Comment by jjavierdguezas** [comment, marked as answer]
@manojVivek @usama8800 default devices can't be deleted right now, but custom ones **can**:

![image](https://user-images.githubusercontent.com/13673443/186405176-8aabd1b1-2f1c-40c9-854b-8184d6b827a8.png)

1- Open the Device Manager
2- Move the custom device to the inactive list
3- Hover over the far right of the device card
4- A button to remove the device will appear

**Comment by manojVivek** [comment]
Oh yeah, sorry, you're right. I totally missed it. 😅
